DAOS-Support.md

```diff
@@ -163,3 +155,57 @@ Number of items that exist in both directories and have different types: 0 (Src:
 Number of items that exist in both directories and have the same content: 1 (Src: 1 Dest: 1)
 Number of items that exist in both directories and have different contents: 0 (Src: 0 Dest: 0)
 ```
+
+## DAOS Serialization and Deserialization Examples
+
+daos-serialize and daos-deserialize can be used on any type of DAOS container.
+They are DAOS only tools that require HDF5.
+
+daos-serialize will serialize a DAOS container to an HDF5 file. Depending on
+the amount of data, multiple files may be written for each rank specified in the job.
+
+daos-deserialize will deserialize or restore the HDF5 files written into a new
+DAOS container, and a pool UUID to deserialize the data to must be specified.
+
+#### Example One
+
+Show the serialization of a DAOS container
+
+```shell
+$ mpirun -np 3 daos-serialize -v /$pool1/$cont1
+Serializing Container to 0-7bf8037d-823f-4fa5-ac2a-c2cae8f81f57
+Serializing Container to 1-7bf8037d-823f-4fa5-ac2a-c2cae8f81f57
+Serializing Container to 2-7bf8037d-823f-4fa5-ac2a-c2cae8f81f57
```

Comment:
@daltonbohning hmm I think maybe I could use <rank>_ I prefer to use it as a prefix though, because it is easier to specify each one if you are adding the files one by one to the command line with tab completion.

Reply:
@daltonbohning I can also add "rank"

Reply:
@dsikich I'm not set on any way in particular, so we can always revisit this later if needed